llm.el

```diff
@@ -128,6 +131,52 @@ REQUIRED is whether this is required or not."
   type
   required)
 
+(cl-defstruct llm-media
+  "Contains media that can be sent as part of an interaction.
```

Comment:
Note that this should be used only if the `image-input` , `audio-input` or `video-input` capability is on (except for models that are as yet unknown to the llm package).

Also, please note these possible capabilities (which should already be there) in the `llm-capabilities` method.

Reply:
I'll add these to `llm-capabilities` this evening.

Reply:
Yeah, they should be added from the model capabilities.  But don't throw an error if it isn't supported, for now `llm-capabilities` is advisory only, and we can't know about every model, so we are cautious with it.

Reply:
I've added now, and updated the docstring.